### PR TITLE
Rename remaining Atom Shell references

### PR DESCRIPTION
**Pull request by kevinsawicki**
This pull request removes all references to the old Atom Shell name from the repo.

After seeing #5670 it seems worth changing the remaining helper functions and environment variables to say Electron instead of Atom Shell to prevent confusion.

/cc @electron/maintainers

### CODE REVIEW

**Comment by mnquintana** [comment]
What do you think about also renaming all the file / directory / class / namespace names in the repo that still include `atom`?

**Comment by kevinsawicki** [comment]
> What do you think about also renaming all the file / directory / class / namespace names in the repo that still include atom?

I think that should be done as well eventually in probably a few different pull requests since that is much more extensive.

**Comment by paulcbetts** [comment]
@kevinsawicki This is highly likely to break people's CI setups (including our own) - maybe support both variables for a time then yank them later?

**Comment by kevinsawicki** [comment]
> This is highly likely to break people's CI setups (including our own) - maybe support both variables for a time then yank them later?

Just because of the S3 variables? Or something else here?

**Comment by kevinsawicki** [comment]
>  maybe support both variables for a time then yank them later?

Added warnings for `ATOM_SHELL_` env var usage 👍

**Comment by paulcbetts** [comment]
> Just because of the S3 variables? Or something else here?

I was referring to the env var names in general

**Comment by kevinsawicki** [comment]
> we can rename the IPC constant names w/o deprecations though

These were already all renamed in #5061

File: script/lib/util.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def atom_gyp():
     return obj['variables']
 
 
-def get_atom_shell_version():
+def get_electron_version():
   return 'v' + atom_gyp()['version%']
```
Comment (zcbenz) [inline script/lib/util.py]: The `atom_gyp` can also be renamed.

Comment (kevinsawicki) [inline script/lib/util.py]: Done 👍

**Comment by zcbenz** [comment]
👍

**Pull request merged**